base.py

```diff
@@ -498,10 +498,21 @@ def update_time(scene):
 class MBDynReadLog(bpy.types.Operator):
     """ Imports MBDyn nodes and elements by parsing the .log file """
     bl_idname = "animate.read_mbdyn_log_file"
-    bl_label = "MBDyn .log file parsing"
```

Comment:
I get it now. I didn't really get you the first time.
That seems like a far more simpler solution. I assume the default option according to you is `HIDE`?